### DISCUSSION TITLE
是否应该添加本地搜索功能？

### DISCUSSION THREAD

**Discussion by lxchapu** [lxchapu/astro-gyoza#4] [General]
我正在考虑是否应该使用 [fuse.js](https://www.fusejs.io/) 实现搜索功能，而不是使用 DocSearch。

基于以下原因：
- DocSearch 需要单独去申请。
- 我并不喜欢它的搜索框样式，而且也不方便自定义。

**Comment by Linux-qitong** [comment]
个人认为可添加本地搜索和网络搜索，选择权交给用户
